### PR TITLE
Hvg

### PR DESCRIPTION
**Pull request by Koncopd**
Added test for filter_genes_dispersion, updated docs

### CODE REVIEW

**Comment by Koncopd** [comment]
Just don't understand why build fails...

**Comment by falexwolf** [comment]
It seems to be a stalled build in CI. Something to do with a URL request... if the tests run through on your end, everything should be fine. Do they?

File: docs/release_notes.rst [hunk 1]
```diff
@@ -7,12 +7,18 @@
 .. role:: smaller
 
 
+On master, :small:`November 5, 2018`
```
Comment (falexwolf) [inline docs/release_notes.rst:10]: No need for a new entry, here, just update the date remove the comma and the statement about "deprecation"; add your name with "thanks to"

**Comment by falexwolf** [comment]
Thank you for this!

**Pull request merged**